reference_points_picker.py

```diff
@@ -64,7 +64,7 @@ def _configure_logger(args: argparse.Namespace) -> logging.Logger:
     else:
         log_dir = Path(args.log_dir)
 
-    log.add_file_handler(log_dir=log_dir, level=log_level_file)
+    log.add_file_handler(log_file=log_dir, level=log_level_file)
```

Comment:
Until now, a directory has been handed over to the logger. Should we add a file name or use the default filename?

Reply:
Yes, let us implement the same behavior. I simply forgot to add the CLI options `--logfile` and `logfile_overwrite` for the reference points picker.